src/libraries/pretest.proj

```diff
@@ -94,4 +91,7 @@
       TargetFilePrefixes="ref/;runtimes/"
       RootAttributes="@(FrameworkListRootAttributes)" />
   </Target>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" />
```

Comment:
The props go at the top of the file, the targets at the end 😄

Reply:
ohh oops, thanks!